wp-content/themes/wp-starter/src/styles/main.css

```diff
@@ -22,3 +22,5 @@
 @import '../../blocks/text-image/style.css';
 @import '../../blocks/text-icon-cards/style.css';
 @import '../../blocks/video-embed/style.css';
+
+/*DO NOT REMOVE - Everything below this line is automatically generated*/
```

Comment:
Is it possible to import a different CSS file here, instead of having auto-generated styles added to this one? That way the entire file can be "off limits" instead of part of one that might need to be edited.

Reply:
That is a great idea!